qtconsole/jupyter_widget.py

```diff
@@ -292,22 +292,30 @@ def _started_channels(self):
     def _process_execute_error(self, msg):
         """Handle an execute_error message"""
         content = msg['content']
-        traceback = '\n'.join(content['traceback']) + '\n'
-        if False:
-            # FIXME: For now, tracebacks come as plain text, so we can't use
-            # the html renderer yet.  Once we refactor ultratb to produce
-            # properly styled tracebacks, this branch should be the default
-            traceback = traceback.replace(' ', '&nbsp;')
-            traceback = traceback.replace('\n', '<br/>')
-
-            ename = content['ename']
-            ename_styled = '<span class="error">%s</span>' % ename
-            traceback = traceback.replace(ename, ename_styled)
-
-            self._append_html(traceback)
+
+        if content['ename'] == 'KeyboardInterrupt':
```

Comment:
No, I don't think special handling of certain Python exceptions is the right thing to do. If KeyboardInterrupt in IPython's run shouldn't display a traceback, that is probably a change that should be made in IPython, rather than the QtConsole.